demo/docker/flink/flink-conf.yaml

```diff
@@ -73,6 +73,11 @@ state.backend: filesystem
 state.checkpoints.dir: file:///opt/flink/data/checkpoints
 state.savepoints.dir: file:///opt/flink/data/savepoints
 
+state.backend.rocksdb.metrics.estimate-num-keys: true
+state.backend.rocksdb.metrics.estimate-live-data-size: true
+#We can have many jobs per cluster, in such setting managed memory is not easy to tune
+state.backend.rocksdb.memory.managed: false
```

Comment:
Please add e.g. `state.backend.rocksdb.writebuffer.size: 1000MB` + comment.

Reply:
I increased to 256m and added comment, I think in docker-compose setup it should be enough...